tests/commands/load/test_load_case_cmd.py

```diff
@@ -21,6 +22,54 @@ def test_load_case(mock_app, institute_obj, case_obj):
     assert os.path.exists(load_path)
 
     # Test command to upload case using demo resources:
-    result =  runner.invoke(cli, ['load', 'case', load_path ])
+    result = runner.invoke(cli, ['load', 'case', load_path ])
     assert result.exit_code == 0
     assert store.case_collection.find().count() == 1
+
+
+
+
+def test_load_case_KeyError(mock_app, institute_obj, case_obj):
```

Comment:
Could it be that the sed command creates an extra file at the end of the test. After pytest on this file I end up with an extra config file: 
![image](https://user-images.githubusercontent.com/28093618/63915536-9ea26b80-ca36-11e9-9356-e41dac4bcfff.png)

Maybe a better way to handle editing the config file would be making a copy of it on a temporary file?
See here: https://docs.python.org/3/library/tempfile.html and here: https://stackoverflow.com/questions/6587516/how-to-concisely-create-a-temporary-file-that-is-a-copy-of-another-file-in-pytho

Reply:
Added use of temp file instead:

```
    temp_conf = os.path.join(tempfile.gettempdir(), 'temp.conf')
    shutil.copy2(load_path, temp_conf)
    sed_change = "sed -i -e 's/sample_id/SAMPLE_ID/g' "
    os.system(sed_change + temp_conf)
```